mozfldp/server.py

```diff
@@ -24,74 +19,120 @@ class ServerFacade:
     This class acts as a facade around the functions in
     `simulation_util` to provide a consistent interface to load data
     into a classifier.
+
+    Args: 
+        coef: initial coefficients to initialize the server with
+        intercept: initial intercepts to initialize the server with
+        num_client: number of clients that are available
+        client_fraction: (C in algorithm) - fraction of clients that we want to use in a round
     """
 
-    def __init__(self):
-        self._classifier = SGDClassifier(loss="hinge", penalty="l2")
+    def __init__(self, coef, intercept, num_client, client_fraction):
+        self._coef = coef
+        self._intercept = intercept
 
-        self._buffer = {"some_client_uuid_1": [], "some_client_uuid_2": []}
+        self._num_client = num_client
+        self._client_fraction = client_fraction
+
+        self._client_coefs = []
+        self._client_intercepts = []
+        self._num_samples = []
 
     def ingest_client_data(self, client_json):
```

Comment:
Another relatively minor stylistic point: it would be nice to use keyword args to pass in the individual components of the client update, so that the details of request handling (ie unwrapping JSON) are encapsulated in the `ingest_client_data` API function below.